### PR TITLE
Added animals route with data

### LINKED ISSUE #78: Adding animals data and pictures
**Describe the feature**
Adding JSON data for animals representing different animals with their breeds and images

**Are you working on this feature? (Yes)**

### PR DESCRIPTION
**Pull request by Samridhi-98**
# Related Issue
- closes #78 

# Proposed Changes
- Added `animaldata.json` in `data` folder
- Added `animals.ts` in `route` folder
- Updated `app.ts` with animal route

# Checklist
- [x] Tests
- [x] Translations

### CODE REVIEW

**Comment by Samridhi-98** [comment]
@ShauryaAg  @paramsiddharth  Please review my PR.

File: src/api/routes/animals.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        console.log(data);
+        res.status(200).json(data);
+    }
+    catch (err) {
+        console.log(err);
+        res.status(500).json({
+            error: err,
+        });
+    }
+});
```
Comment (ShauryaAg) [inline src/api/routes/animals.ts:18]: Can we please add a route to get a specific animal?

**Review by ShauryaAg — CHANGES_REQUESTED** [review]

File: src/api/routes/animals.ts (hunk 1 above)
Comment (Samridhi-98) [reply src/api/routes/animals.ts:18]: Okay, will do that.

**Comment by Samridhi-98** [comment]
@ShauryaAg  @paramsiddharth  Done! please review my PR

**Review by ShauryaAg — APPROVED** [review]

**Pull request merged**